ts/parts/Axis.ts

```diff
@@ -5603,8 +5596,8 @@ class Axis implements AxisComposition {
                     this.closestPointRange,
                     true
                 );
-            } else if (this.isLog) {
-                tickPositions = axis.getLogTickPositions(
+            } else if (this.isLog && axis.logarithmic) {
```

Comment:
Can we remove `isLog` and just simply test for `axis.logarithmic`? That also means when updating an axis from `log` to `lin` we need to delete `axis.logarithmic`.

Reply:
No as isLog is different in the sense, that it indicates, if the functionality is active and not just loaded/available.

Reply:
That's what I mean.... Should `axis.logarithmic` only be present when the functionality is active? Then we would need some kind of construct like

```js
if (type === 'logarithmic') {
    this.logarithmic = new LogarithmicAxisAdditions(this);
}
```

... and then this object would be deleted again if the axis is updated to non-logarithmic.

Reply:
I see. With the last two tries, I did not thought of deleting it later on. We can try that tomorrow.